### ISSUE TITLE
Using rtv causes exception

### ISSUE THREAD

**Issue by 2l47** [sapphonie/sourcemod-nativevotes-updated#26]
[Build d45ba55](https://github.com/sapphonie/sourcemod-nativevotes-updated/releases/tag/workflow-build51)
```
L 08/18/2022 - 21:40:23: [nativevotes_mapchooser.sp.smx] Starting map vote because outside request
L 08/18/2022 - 21:40:23: [SM] Exception reported: Invalid key value handle 1 (error 1)
L 08/18/2022 - 21:40:23: [SM] Blaming: nativevotes.smx
L 08/18/2022 - 21:40:23: [SM] Call stack trace:
L 08/18/2022 - 21:40:23: [SM]   [0] KeyValues.GetNum
L 08/18/2022 - 21:40:23: [SM]   [1] Line 180, nativevotes/data-keyvalues.sp::Data_GetResultCallback
L 08/18/2022 - 21:40:23: [SM]   [2] Line 1979, nativevotes.sp::Native_SetResultCallback
L 08/18/2022 - 21:40:23: [SM]   [4] NativeVote.VoteResultCallback.set
L 08/18/2022 - 21:40:23: [SM]   [5] Line 622, nativevotes_mapchooser.sp::InitiateVote
L 08/18/2022 - 21:40:23: [SM]   [6] Line 1502, nativevotes_mapchooser.sp::Native_InitiateVote
L 08/18/2022 - 21:40:23: [SM]   [8] InitiateMapChooserVote
L 08/18/2022 - 21:40:23: [SM]   [9] Line 376, nativevotes_rockthevote.sp::StartRTV
L 08/18/2022 - 21:40:23: [SM]   [10] Line 337, nativevotes_rockthevote.sp::AttemptRTV
L 08/18/2022 - 21:40:23: [SM]   [11] Line 256, nativevotes_rockthevote.sp::OnClientSayCommand_Post
```

**Comment by sapphonie** [comment]
Allegedly fixed in #27

**Comment by 2l47** [comment]
fixed
